### PR TITLE
interactive examples

### PR DESCRIPTION
**Pull request by willmcgugan**
Adds interactive examples to the docs.

It's a bit of an experiment. Seems to work well enough.

Will be interesting to see how it copes if anything goes viral...

https://github.com/Textualize/textual/assets/554369/6d4bfc91-0cbf-4956-aed5-17eb6ba555e6

### CODE REVIEW

File: docs/tutorial.md [hunk 1]
```diff
@@ -31,6 +31,18 @@ Here's what the finished app will look like:
 ```{.textual path="docs/examples/tutorial/stopwatch.py" title="stopwatch.py" press="tab,enter,tab,enter,tab,enter,tab,enter"}
 ```
 
+### Try it out!
+
+The following is *not* a screenshot, but a fully interactive Textual app running in your browser.
+
+
+!!! textualize "Try in Textual-web"
```
Comment (rodrigogiraoserrao) [inline docs/tutorial.md:39]: You are using `!!!` here and `???` in the other examples. Is this OK?

**Review by rodrigogiraoserrao — COMMENTED** [review]
“Try in Textual-web” sounds a bit awkward but it may be a limitation of my English. I'd personally prefer “Try it with textual-web”, or something else entirely like “Try it” or “Interactive demo”.

I'm excited to give this a go!

File: docs/tutorial.md (hunk 1 above)
Comment (willmcgugan) [reply docs/tutorial.md:39]: The `???` puts it in an expandable container. For the tutorial I wanted it to already be expanded.

**Comment by willmcgugan** [comment]
> I'm excited to give this a go!

It should work if you build it locally.

File: docs/tutorial.md (hunk 1 above)
Comment (rodrigogiraoserrao) [reply docs/tutorial.md:39]: Understood.

**Comment by rodrigogiraoserrao** [comment]
> > I'm excited to give this a go!
> 
> It should work if you build it locally.

I confirm it did work.

**Pull request merged**